test-e2e/tests/live_channel_test.ts

```diff
@@ -78,7 +78,8 @@ Scenario('I can navigate to live channels from the header', ({ I }) => {
   I.see('On Channel 1', locate('div').inside(videoDetailsLocator));
 });
 
-Scenario('I can watch the current live program on the live channel screen', async ({ I }) => {
+// eslint-disable-next-line codeceptjs/no-skipped-tests
```

Comment:
Would be good to at least put a "TODO" here to remind us to re-enable this. I've also done stuff with date checks to temporarily pass/skip the test in code until X date, then start testing it again (that way we'll see it start failing if the stream doesn't get fixed)

Reply:
I have created a ticket in the backlog and added TODO to the skipped test. I will ask the team to create a stream when we get OTT params feature in `prd`.